tests/view/items/test_items.py

```diff
@@ -29,7 +29,7 @@ def test_list(client):
 
 def test_create(client):
     """ Test create is okay """
-    assert client.get("/items/create").status_code == HTTPStatus.OK
+    assert client.post("/items/create").status_code == HTTPStatus.OK
```

Comment:
@paulodamaso let's improve test to test that item is really creating during the request. Now test is poor.